src/main/java/net/pms/dlna/RootFolder.java

```diff
@@ -1477,8 +1477,8 @@ public void notify(String filename, String event, FileWatcher.Watch watch, boole
 						RealFile rf = new RealFile(file);
 						rf.setParent(rf);
 						rf.getParent().setDefaultRenderer(RendererConfiguration.getDefaultConf());
-
 						rf.resolveFormat();
+						rf.syncResolve();
```

Comment:
@SubJunk It's called from ```DLNAResource.run()```. I don't know in which contect ```DLNAResource``` is used as a runnable, but as the whole parsing/resolving scheme is very hard to get an overview of. It seems to me that ```run()``` is called very often, just place a breakpoint in the method and start UMS and you'll see. In fact, it's called so often that I don't know if this is the correct place to trigger the OpenSubtitles call. If you look at ```RealFile.resolve()``` (which is called by ```syncResolve()```) you'll see that this is where the ```DLNAMediaInfo``` instance is called, where the cache is checked and where the parsing is initialized. It's very central I'd say, but since the OpenSubtitles lookup is only relevant for real files, maybe the call should be moved to ```RealFile.resolve()```.

My point was that in trhe above method, without calling ```syncResolve()``` ```rf``` was never parsed - which kind of defeats the purpose of detecting if a file is being added. I assume that you don't only want to do the OpenSubtitles stuff when a file is added, but a full parsing so that all of that can be added to the database?